sklearn/cluster/tests/test_mean_shift.py

```diff
@@ -93,21 +99,24 @@ def test_parallel():
         random_state=11,
     )
 
+    X = X.astype(global_dtype, copy=False)
+
     ms1 = MeanShift(n_jobs=2)
     ms1.fit(X)
 
     ms2 = MeanShift()
     ms2.fit(X)
 
-    assert_array_almost_equal(ms1.cluster_centers_, ms2.cluster_centers_)
+    assert_allclose(ms1.cluster_centers_, ms2.cluster_centers_)
```

Comment:
I would as well check that the `dtype` are the same.

Reply:
done